src/app/components/Recommendations/index.tsx

```diff
@@ -0,0 +1,103 @@
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx, useTheme } from '@emotion/react';
+
+import SectionLabel from '#psammead/psammead-section-label/src';
+import SkipLinkWrapper from '#components/SkipLinkWrapper';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import useViewTracker from '#app/hooks/useViewTracker';
+import { Recommendation } from '#app/models/types/onwardJourney';
+import RecommendationsItem from './RecommendationsItem';
+import styles from './index.styles';
+
+const eventTrackingData = {
+  componentName: 'midarticle-mostread',
+};
+
+const Recommendations = ({ data }: { data: Recommendation[] }) => {
+  const { recommendations, mostRead, script, service, dir } =
+    useContext(ServiceContext);
+
+  const viewEventTracker = useViewTracker(eventTrackingData);
+
+  const {
+    palette: { GREY_2 },
+  } = useTheme();
+
+  const labelId = 'recommendations-heading';
+
+  const a11yAttributes = {
+    role: 'region',
+    'aria-labelledby': labelId,
+  };
+
+  const { hasMostRead } = mostRead || {};
+  const { skipLink, header } = recommendations || {};
+
+  const { text, endTextVisuallyHidden } = skipLink || {
+    text: 'Skip %title% and continue reading',
+    endTextVisuallyHidden: 'End of %title%',
+  };
+
+  if (!hasMostRead || !data?.length) return null;
```

Comment:
Should we move this closer to the top, so that we're exiting before running additional code?